Tests/_CryptoExtrasTests/Utils/BytesUtil.swift

```diff
@@ -70,8 +70,9 @@ extension DataProtocol {
                         offset += 1
                     }
                 }
-                return offset
+                count = offset
             }
+            return String(bytes: bytes, encoding: .ascii)!
```

Comment:
No need to use this Foundation API here, we can instead use `String(decoding: bytes, as: UTF8.self)`.

Reply:
Thanks! Updated.